sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -13,9 +13,9 @@
     <Description>Provides management capabilities for Microsoft Azure Iot Hub.</Description>
     <AssemblyTitle>Microsoft Azure Iot Hub Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-    <Version>3.0.1</Version>
+    <Version>4.0.0</Version>
```

Comment:
Major version update because 2 APIs changed to require request body (or now in the form of a class)